ecs-init/engine/engine.go

```diff
@@ -165,10 +167,36 @@ func (e *Engine) load(image io.ReadCloser, err error) error {
 	return e.downloader.RecordCachedAgent()
 }
 
+// retryDelay returns an exponential backoff for delaying the service restart.
+// delay will look something like this:
+//     Retry 1: 2.162s
+//     Retry 2: 7.548s
+//     Retry 3: 14.776s
+//     Retry 4: 16.944s
+//     Retry 5: 34.592s
+//     Retry 6: 1m0.425s
+//     Retry 7: 1m0.456s
+//     Retry 8: 1m0.694s ...etc...
+func retryDelay(retryCount int) time.Duration {
+	minTime := 1000
+	maxTime := time.Second * 60
```

Comment:
can you make 1000 and 60 * time.Second as consts in this package?

Reply:
I don't foresee these values getting used anywhere else so I'm not sure making constants is necessary here

Reply:
this is done now